src/components/Details.tsx

```diff
@@ -47,8 +49,11 @@ type ElementProps = {
 type StyleProps = {
   layout?: 'column' | 'row' | 'rowColumns' | 'grid' | 'stackColumn';
   justifyItems?: 'start' | 'end';
+  // I don't know why we're getting false positives for these props
```

Comment:
[non-blocking] let's make a ticket to investigate this

Reply:
I think it's just a straightforward shortcoming of the eslint rule. Those props are technically unused but we're also using the type as a way of typing the details items which need key and subitem. I think it's fine here.